.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Describe what is this change, why is it needed. Mention the issue this PR resolves (`Closes ###`).
+
+This change:
+
+- [ ] Fixes a bug
+- [ ] Adds new feature
+- [ ] Enhancement
```

Comment:
@stoader Could you define this briefly? How does this relate to `new feature`? Why is this important (other than for reporting/invoicing working hours spent by type of effort)?

Reply:
It's just for reporting. I'm fine with removing it.